modules/eco-poison-pill-operator-installation-cli.adoc

```diff
@@ -64,7 +61,7 @@ metadata:
     name: poison-pill-manager
     namespace: poison-pill
 spec:
-    channel: alpha
+    channel: stable
     name: poison-pill-manager
     source: redhat-operators
     sourceNamespace: openshift-marketplace
```

Comment:
![image](https://user-images.githubusercontent.com/58551443/155094735-d589c5ba-6c91-412a-aecb-afd74ad1688f.png)
version of PP in 4.9 will be 0.2.0